odoo/addons/base/models/ir_actions.py

```diff
@@ -566,7 +567,8 @@ def _default_update_path(self):
                        help="Write Python code that the action will execute. Some variables are "
                             "available for use; help about python expression is given in the help tab.")
     # Multi
-    child_ids = fields.Many2many('ir.actions.server', 'rel_server_actions', 'server_id', 'action_id',
+    parent_id = fields.Many2one('ir.actions.server', string='Parent Action', ondelete='cascade')
+    child_ids = fields.One2many('ir.actions.server', 'parent_id', copy=True, domain=lambda self: f"[{', '.join(self._get_linkable_childs_domain_parts())}]",
```

Comment:
Make it return a `Domain` or `str`. A little bit like it is done in `_description_domain` of relational fields.
```suggestion
    child_ids = fields.One2many('ir.actions.server', 'parent_id', copy=True, domain=lambda self: self._get_children_domain(),
```